auth/loginHandler.js

```diff
@@ -0,0 +1,22 @@
+const bcrypt = require("bcrypt");
+
+const getUserByemail = require("../controllers/users");
+const issueToken = require("./issueToken");
+
+const loginHandler = async (email, incomingPassword) => {
+    const user = await getUserByemail(email);
+    if (!user) {
+        throw { code: 404, msg: "User not found!!!" };
```

Comment:
tutaj zamiast throw, możemy użyć instrukcji return albo zwrócić nowy error np. 
`
throw new Error({ code: 404, msg: "User not found!!!" })`

taka sama sytuacja jest kilka linijek niżej

Reply:
Poprawione